styles/preview.css

```diff
@@ -47,6 +47,10 @@
     text-align: left;
 }
 
+#color_test {
```

Comment:
should we move this to less one day?

Reply:
@srjfoo and I were chatting about this too. At some point some of this should move into `page_interface.less`, but perhaps not all of it.